pkg/controllers/provisioning/v1alpha1/allocator/bind.go

```diff
@@ -54,8 +52,12 @@ func (b *Binder) Bind(ctx context.Context, provisioner *v1alpha1.Provisioner, no
 }
 
 func (b *Binder) decorate(provisioner *v1alpha1.Provisioner, node *v1.Node) {
-	node.ObjectMeta.Labels = map[string]string{
-		KarpenterNodeLabelKey: fmt.Sprintf("%s-%s", provisioner.Name, provisioner.Namespace),
+	node.Labels = map[string]string{
```

Comment:
does this need to be `node.ObjectMeta.Labels`?

Reply:
It's using embedded struct field inference. They're the same.